handlers.go

```diff
@@ -215,6 +216,8 @@ func (a *autographer) handleSignature(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 		requestedSignerConfig := requestedSigner.Config()
+		a.stats.Incr("signer.requests", []string{"keyid:" + requestedSignerConfig.ID, "user:" + userid, "request_id:" + rid, usedDefaultSignerTag(sigreq)}, 1.0)
```

Comment:
What is the cardinality of each tag? I think `keyid` and `user` will be a limited list (because it's just the value from the config file, right) but I think `request_id` will blow up influx storage and prometheus costs.

Reply:
Combining the signer and user ID it's [38 on production](https://lookerstudio.google.com/u/0/reporting/69aab939-4946-4157-b33c-252c82780cb8/page/p_xpwq5dyxmd/edit). Individually it's 32 user ID and 37 signer ID. 

@jbuck that's a good point about `request_id`.

Reply:
Yeah, I was thinking of tracing systems, not metric systems, when adding that request_id. Removed.